openapi.yaml

```diff
@@ -164,704 +128,833 @@ paths:
-        - birthDate
-        - sex
+      firstName:
+        type: string
+        description: First name of person
+        example: John
+      lastName:
+        type: string
+        description: Last name of person
+        example: Smith
+      middleName:
+        type: string
+        description: Middle name of person
+        example: Adam
+      birthDate:
+        type: string
+        format: date
+        description: Birthday of person
+        example: '1970-1-21'
+      citizen:
+        type: object
         properties:
-          name:
-            properties:
-              firstName:
-                type: string
-                description: First name
-              lastName:
-                type: string
-                description: Last name
-              middleName:
-                type: string
-                description: Middle name
-          birthDate:
-            type: string
-            format: date
-            description: Birth date
-            example: '1995-01-20'
-          citizen:
+          code:
             type: string
             enum: [FN, N, R, S, C]
             description: |
-              Person's citzen code
-              - 'FN' - Foreign National
-              - 'N' - Non Resident Alien
-              - 'R' - Resident Alien
-              - 'S' - Substantial Presence Alien
-              - 'C' - Citizen
-          sex:
+              Person's citizen code
+              - `FN` - Foreign National
+              - `N` - Non Resident Alien
+              - `R` - Resident Alien
+              - `S` - Substantial Presence Alien
+              - `C` - Citizen
+          description:
             type: string
-            enum: [M, F, N]
-            description: |
-              Person's sex code
-              - 'M' - Male
-              - 'F' - Female
-              - 'N' - Non-specified
-  PersonResultObject:
+            description: Description field for the code
+      sex:
+        type: string
+        enum: [M, F, N]
+        description: |
+          Person's sex code
+          - `M` - Male
+          - `F` - Female
+          - `N` - Non-specified
+  PersonAttributes:
+    type: object
     properties:
-      links:
-        $ref: '#/definitions/SelfLink'
-      data:
+      displayFirstName:
+        type: string
+        description: Preferred first name
+        example: John
+      displayMiddleName:
+        type: string
+        description: Preferred middle name
+        example: Adam
+      displayLastName:
+        type: string
+        description: Preferred last name
+        example: Smith
+      onid:
+        type: string
+        description: ONID username
+        example: smithj
+      isConfidential:
+        type: boolean
+        description: If true, the person has opted in to being required to be listed as confidential
+      isCurrentStudent:
+        type: boolean
+        description: True if person is a current student
+      isCurrentEmployee:
```

Comment:
Agreed. `isCurrentStudent` does the same thing. Think we should return `studentStatus` there as well?

Reply:
I think `isCurrentStudent` can only be either`Y` and `N` (or maybe `null`) so it will make sense to keep it as boolean. We probably just wanna rename it to `currentStudentInd` for the consistency.

Reply:
```
New Student
Inactive Student
Withdrawn Student
Continuing
Readmit/Old Returning
Summer Only Student
Continuing Higher Education
Auditor
Non-matriculated student
```
There are other options.

Reply:
This is already returned in students so we will keep it as is.